build.gradle

```diff
@@ -54,11 +56,14 @@ repositories {
 
 
 dependencies {
+    errorprone 'com.google.errorprone:error_prone_core:2.3.1'
     compile 'org.apache.velocity:velocity:1.7'
     compile 'org.yaml:snakeyaml:1.10'
     compile 'com.github.jgraph:jgraphx:v3.3.1.1'
-    compile 'org.projectlombok:lombok:1.16.18'
+    compile 'com.sun.activation:javax.activation:1.2.0'
```

Comment:
## ⚠️ 🔥 ⚠️ Someone should read this licence!!!! ⚠️ 🔥 ⚠️ 

Let's make sure we're not coupling ourself to a licence we don't want.
https://search.maven.org/artifact/com.sun.activation/javax.activation/1.2.0/jar

Reply:
Looks like Oracle's standard GPL with classpath exception.

Why is this required, though?

Reply:
@SamCarlberg Are you allowed to re-package and publish releases with oracle's jar's in them with their GPL with classpath exception licence?

Reply:
It's deprecated in JDK 9, and the module isn't loaded by default. This seemed like the easiest and most future proof of oracle's migration suggestions.

Reply:
Ah, looks like it's used for DnD. Looks nontrivial to migrate to something else so this is fine with me

Reply:
DnD?

Reply:
Drag-and-drop

Reply:
You need to re-distribute this application with the oracle licence if you add this dependency. Just FYI.